golang-github-prometheus-prometheus-image/golang-github-prometheus-prometheus-image.kiwi.j2

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  This file is generated, don't manually modify this file!
+-->
+<image name="golang-github-prometheus-prometheus-image" schemaversion="6.9" xmlns:suse_label_helper="com.suse.label_helper">
```

Comment:
Do we have some CI relying on the package name _golang-github-prometheus-prometheus_ and do we have to use it for directory name as well? _prometheus-image_ looks much nicer.

Reply:
currently I did find any convention, so went ahead with package name. 
But I agree that `prometheus-image` is better